errors.yaml

```
@@ -48,4 +48,8 @@ errors:
     description: "There was an error in the search."
   - code: unsupported_media_type
     description: "The payload content type is not supported by MeiliSearch. Currently, MeiliSearch supports only JSON payloads."
+  - code: backup_already_in_process
+    description: "A backup is already in process and a new one can't be triggered until previous backup creation is not finished."
+  - code: backup_process_failed
+    description: "An error occured during backup process, task aborted."
```

```suggestion
    description: "An error occured during backup creation process, task aborted."
```

Can this process be something else than a creation process? Like a downloading process maybe?